tests/client/fixtures/rights-unicode-filepath.csv

```diff
@@ -1,2 +1,2 @@
 file,basis,status,determination_date,jurisdiction,start_date,end_date,terms,citation,note,grant_act,grant_restriction,grant_start_date,grant_end_date,grant_note,doc_id_type,doc_id_value,doc_id_role
-objects/たくさん directories/need sanitization/checking here/evélyn's photo.jpg,copyright,cop status,2001-01-01,cop juris,2002-02-02,2003-03-03,,,cop note,cop act,Allow,2004-04-04,2005-05-05,cop grant note,cop type,cop value,cop role
+objects/たくさん directories/need_name_change/checking here/evélyn's photo.jpg,copyright,cop status,2001-01-01,cop juris,2002-02-02,2003-03-03,,,cop note,cop act,Allow,2004-04-04,2005-05-05,cop grant note,cop type,cop value,cop role
```

Comment:
This path should use spaces between the words instead of underscores, like the original.